common/djangoapps/course_modes/helpers.py

```diff
@@ -13,6 +15,43 @@
 DISPLAY_AUDIT = "audit"
 DISPLAY_PROFESSIONAL = "professional"
 
+COURSE_INFO_PAGE = "COURSE_INFO"
+SUCCESS_ENROLL_PAGE = "ENROLL_SUCCESS"
+
+
+def get_mktg_url(page):
```

Comment:
Please remove this as well, and use edX provided helpers, I've added an entry to the [Integrate the Edraak marketing site with Eucalyptus](https://app.asana.com/0/256944342521670/303375741516935) task on Asana to address this later.